hmda/src/test/scala/hmda/validation/rules/lar/validity/2018/V619_2Spec.scala

```diff
@@ -4,11 +4,10 @@ import hmda.model.filing.lar.LarGenerators._
 import hmda.model.filing.lar.LoanApplicationRegister
 import hmda.validation.rules.EditCheck
 import hmda.validation.rules.lar.LarEditCheckSpec
-import hmda.validation.rules.lar.validity._2018.V619_2
-
+import hmda.validation.rules.lar.validity.common.V619_2
 
 class V619_2Spec extends LarEditCheckSpec {
```

Comment:
Can these tests be combined into a single test file in `...rules/lar/validity/common/V619_2Spec.scala` to check that all the filing years work as expected for V619_2? That way we won't have as much repetitive code to test each year

Reply:
Sure thing good catch